### PR TITLE
add process documentation

### PR DESCRIPTION
**Pull request by scoopex**
Write documentation for the process, especially for colleagues which do not use git and Github in their daily work very often.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,94 @@
...
+1. Commit and push with signoffs
+   ```
+   git status
+   git diff
+   git add <file> ... <file>
+   git status
+   git commit -s -a
+   gut push 
+   ```
+2. Optional: Testing complexer changes
```
Comment (fkr) [inline README.md:63]:
```suggestion
2. Optional: Testing more complex changes
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,94 @@
...
+* [The productive SCS site](https://scs.community/de/)
+* [The staging area](https://staging.scs.community/de/)
+
+## The basic editorial workflow
+
+The website data is deployed by the following process:
+
+![Website workflow](_assets/images/readme/website_workflow.drawio.svg)
+
+The following steps describe the basic workflow how to add, change and remove content.
```
Comment (fkr) [inline README.md:29]:
```suggestion
The following steps describe the basic workflow on how to add, change and remove content.
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,94 @@
...
+   git clone git@github.com:SovereignCloudStack/website.git # you only only the fist time
+   cd website
+   git checkout main
+   git pull
+   git checkout -b feat/<name of your branch> # not not use spaces
+   ``` 
+2. Make your changes with the tool of your choice
+   <details>
+   <summary>Visual Studio Code</summary>
+     - Download and install [Visual Studio Code](https://code.visualstudio.com/) Development Environment on your workstation
```
Comment (fkr) [inline README.md:44]: Why do we advocate for VS Code? Any specific reason?

File: README.md [hunk 4]
```diff
@@ -1,8 +1,94 @@
...
+* [The staging area](https://staging.scs.community/de/)
+
+## The basic editorial workflow
+
+The website data is deployed by the following process:
+
+![Website workflow](_assets/images/readme/website_workflow.drawio.svg)
+
+The following steps describe the basic workflow how to add, change and remove content.
+The description is a little more detailed than it needs to be, as it aims also to provide a reliable guide for non-engineer colleagues.
```
Comment (fkr) [inline README.md:30]: While I understand the reasoning, I think this makes reading this documentation cumbersome (and as such leads to people not reading it ;)
The same methods of working (sign-off etc.) are used for all the repos (to which non-engineer colleagues contribute as well). This leads to duplication of documentation (and as a mid-term result to outdated docs). Of course, this is minor commentary and nothing that should block merging this.

File: README.md [hunk 5]
```diff
@@ -26,3 +112,5 @@ As we're using Bootstrap, please add your custom styling definitions to `assets/
 ## Testing changes on `staging` branch
 
 We're using a dedicated workflow to build a staging preview of our website that subsequently is published with GitHub Pages. To make use of this feature, simply push to `staging`. Please try to keep `staging` in sync with `main`, e.g. by deleting and recreating after testing your recent changes. The staging area can be reached via <https://sovereigncloudstack.github.io/website/>.
```
Comment (fkr) [inline README.md:114]: Shouldn't this be removed / reworded in favour of the addition above?

**Comment by scoopex** [comment]
Thanks for the elaborate comments to this draft PR.

From my current point of view the described procedure seems a bit too inconvenient and I wonder if it wouldn't be better to simplify the process so much technically (e.g. by not using signoffs and automatically generating a branch preview on the staging system) that a few clicks on the github interface would be enough for simple adjustments.
Lets discuss this next week directly.

File: README.md [hunk 6]
```diff
@@ -1,8 +1,94 @@
...
+* [Place for adding further employees](https://github.com/SovereignCloudStack/website/tree/main/_employees)
+* [Place for adding further community profiles](https://github.com/SovereignCloudStack/website/tree/main/_members)
+* [Place for adding further partners](https://github.com/SovereignCloudStack/website/tree/main/_partners)
+* [Place for adding further supporting partners](https://github.com/SovereignCloudStack/website/blob/main/_data/supporter.yml)
+* [Place for adding further providers](https://github.com/SovereignCloudStack/website/blob/main/_data/provider.yml)
+
+Published sites:
+
+* [The productive SCS site](https://scs.community/de/)
+* [The staging area](https://staging.scs.community/de/)
```
[the remark this answers is not in the document]
Comment (scoopex) [reply README.md:21]: No, shortened it.

File: README.md (hunk 5 above)
Comment (scoopex) [reply README.md:114]: Yes. Done

File: README.md (hunk 3 above)
Comment (scoopex) [reply README.md:44]: No, clarified that.

File: README.md (hunk 4 above)
Comment (scoopex) [reply README.md:22]: IMHO is the process to complicated for non-engineers. The experience of the last weeks was that new changes are typically  performed in collaboration

File: README.md [hunk 7]
```diff
@@ -1,8 +1,76 @@
...
+
+![Website workflow](_assets/images/readme/website_workflow.drawio.svg)
+
+The following steps describe the basic workflow how to add, change and remove content.
+The description is a little more detailed than it needs to be, as it aims also to provide a reliable guide for non-engineer colleagues.
+
+1. Create a branch based on the `main` branch
+   (you can also do this using the github user interface)
+   ```
+   git clone git@github.com:SovereignCloudStack/website.git # you only only the fist time
```
Comment (garloff) [inline README.md:27]:
```suggestion
   git clone git@github.com:SovereignCloudStack/website.git # you only need to do this the first time
```

**Review by garloff — APPROVED** [review]
This looks like a clear improvement over the old explanation, so I would merge it.

**Pull request merged**